stdlib/public/core/Algorithm.swift

```diff
@@ -83,33 +85,33 @@ public struct EnumeratedIterator<
   /// Advances to the next element and returns it, or `nil` if no next element
   /// exists.
   ///
-  /// Once `nil` has been returned, all subsequent calls return `nil`.
+  /// - Note: Once `nil` has been returned, all subsequent calls return `nil`.
```

Comment:
Same with this `- Note:`—that sentence doesn't need special attention called to it.